packages/web/src/components/TeeResults.vue

```diff
@@ -194,6 +194,10 @@ const getCostInfos = (program: ProgramData) => {
   // Translate prefix
   prefix = choices.t(prefix)
 
+  // No splitted amounts (non-breakable spaces in texts like '10 000 €')
+  text = text?.replace(/\s0/g, '\u00A00')
```

Comment:
suggestion: this can be move on a generic file (like `helper.ts`) to use again this method if need on other part of code

suggestion:
```js
  const regex = /(?<=\d)\s(?=\d)|(?<=\d)\s(?=\D)/g;
  text = text?.replace(regex, '\u00a0');
```
this is something more generic because it's working with any number (like 1 234) and with/without currency (like 1 234 €)  
What  do you think about this ?

Reply:
note: I created a new function in `utils/helpers.ts` named `consolidateAmounts`.
[Corresponding commit](https://github.com/betagouv/transition-ecologique-entreprises-widget/pull/189/commits/5d5f802a69bcf2c8edbf1a5b0b05307b1edb8a83)